README.md

```diff
@@ -141,26 +166,39 @@ declare(strict_types=1);
 namespace Patchlevel\Enum\Example;
 
 use Patchlevel\Enum\ExtendedEnum;
-use function json_encode;
-use const JSON_THROW_ON_ERROR;
 
 /**
  * @psalm-immutable
- * @method static self UP()
- * @method static self DOWN()
- * @method static self LEFT()
- * @method static self RIGHT()
+ * @method static self CREATED()
+ * @method static self PENDING()
+ * @method static self RUNNING()
+ * @method static self COMPLETED()
  */
-final class Direction extends ExtendedEnum
+final class Status extends ExtendedEnum
 {
-    private const UP = 'up';
-    private const DOWN = 'down';
-    private const LEFT = 'left';
-    private const RIGHT = 'right';
+    private const CREATED = 'created';
+    private const PENDING = 'pending';
+    private const RUNNING = 'running';
+    private const COMPLETED = 'completed';
 }
+```
 
-$directionUp = Direction::UP();
+### __callStatic
```

Comment:
we could add some information that `__callStatic` is using the constants internally.